tests/tests/lightspeed-install.cy.ts

```diff
@@ -26,11 +26,30 @@ describe('Lightspeed related features', () => {
     cy.adminCLI(
       `oc adm policy add-cluster-role-to-user cluster-admin ${Cypress.env('LOGIN_USERNAME')}`,
     );
-    cy.login(
-      Cypress.env('LOGIN_IDP'),
-      Cypress.env('LOGIN_USERNAME'),
-      Cypress.env('LOGIN_PASSWORD'),
-    );
+    // Getting the oauth url for hypershift cluster login
+    cy.exec(
+      `oc get oauthclient openshift-browser-client -o go-template --template="{{index .redirectURIs 0}}" --kubeconfig ${Cypress.env('KUBECONFIG_PATH')}`,
+    ).then((result) => {
+      if (expect(result.stderr).to.be.empty) {
+        const oauth = result.stdout;
+        cy.log(oauth);
+        // Trimming the origin part of the url
+        const oauthurl = new URL(oauth);
+        const oauthorigin = oauthurl.origin;
+        cy.log(oauthorigin);
+        cy.wrap(oauthorigin).as('oauthorigin');
+      }
+    });
+    cy.get('@oauthorigin').then((oauthorigin) => cy.log(oauthorigin));
```

Comment:
I see we're calling `cy.log(oauth)` and calling `cy.log(oauthorigin)` twice. Probably should reduce the calls to `cy.log` to keep the logs easier to read.

Reply:
Agree, my bad. Issues of debugging